RELEASES.md

```diff
@@ -2,6 +2,86 @@
 
 --------------------------------------------------------------------------------
 
+## 0.13.0
+
+Unreleased
+
+### Added
+
+* Lots of documentation of `wasmtime` has been updated. Be sure to check out the
+  [book](https://bytecodealliance.github.io/wasmtime/) and [API
+  documentation](https://bytecodealliance.github.io/wasmtime/api/wasmtime/)!
+
+* All wasmtime example programs are now in a top-level `examples` directory and
+  are available in both C and Rust.
+  [#1286](https://github.com/bytecodealliance/wasmtime/pull/1286)
+
+* A `wasmtime::Linker` type was added to conveniently link link wasm modules
+  together and create instances that reference one another.
+  [#1384](https://github.com/bytecodealliance/wasmtime/pull/1384)
+
+* Wasmtime now has "jitdump" support enabled by default which allows [profiling
+  wasm code on linux][jitdump].
+  [#1310](https://github.com/bytecodealliance/wasmtime/pull/1310)
+
+* The `wasmtime::Caller` type now exists as a first-class way to access the
+  caller's exports, namely memory, when implementing host APIs. This can be the
+  first argument of functions defined with `Func::new` or `Func::wrap` which
+  allows easily implementing methods which take a pointer into wasm memory.
+  [#1290](https://github.com/bytecodealliance/wasmtime/pull/1290)
```

Comment:
It'd be good to mention that this is limited to memory exports right now, and that we hope to replace it in the future with mechanisms that don't require depending on a module's entire memory being exported.